gems/sorbet-runtime/lib/types/private/methods/_methods.rb

```diff
@@ -272,15 +272,8 @@ def self._handle_missing_method_signature(receiver, original_method, callee)
     receiving_method = receiving_class.instance_method(callee)
     if receiving_method != original_method && receiving_method.original_name == original_method.name
       aliasing_mod = receiving_method.owner
-
-      # Note, this logic is duplicated above, make sure to keep changes in sync.
-      if method_sig.check_level == :always || (method_sig.check_level == :tests && T::Private::RuntimeLevels.check_tests?)
-        # Checked, so copy the original signature to the aliased copy.
-        T::Private::Methods.unwrap_method(aliasing_mod, method_sig, original_method)
-      else
-        # Unchecked, so just make `alias_method` behave as if it had been called pre-sig.
-        aliasing_mod.send(:alias_method, callee, original_method.name)
-      end
+      method_sig = method_sig.as_alias(callee)
+      unwrap_method(aliasing_mod, method_sig, original_method)
```

Comment:
Is it worth leaving the `T::Private::Methods` receiver on this?

Reply:
This file seems to be inconsistent in style that way. I have no preference in particular - do you prefer I put it back?

Reply:
If there's no consistent ambient style, I'm fine with leaving the receiver empty.